x: 3
y: 0
z: 2.0
roll: -45
pitch: 0
yaw : 20
dim_box_x: 3
dim_box_y: 1